easybuild/easyblocks/b/bazel.py

```diff
@@ -112,13 +124,43 @@ def configure_step(self):
         # See https://github.com/bazelbuild/bazel/issues/10377
         bazel_args += ' --host_javabase=@local_jdk//:jdk'
 
+        # Link C++ libs statically, see https://github.com/bazelbuild/bazel/issues/4137
+        env.setvar('BAZEL_LINKOPTS', '-static-libstdc++:-static-libgcc')
+        env.setvar('BAZEL_LINKLIBS', '-l%:libstdc++.a')
+
         env.setvar('EXTRA_BAZEL_ARGS', bazel_args)
+        env.setvar('EMBED_LABEL', self.version)
+        env.setvar('VERBOSE', 'yes')
 
     def build_step(self):
         """Custom build procedure for Bazel."""
-        cmd = '%s ./compile.sh' % self.cfg['prebuildopts']
+        # The initial bootstrap of bazel is done in TMPDIR
+        cmd = 'TMPDIR="%s" %s bash -c "set -x && ./compile.sh"' % (self.bazel_tmp_dir, self.cfg['prebuildopts'])
         run_cmd(cmd, log_all=True, simple=True, log_ok=True)
 
+    def test_step(self):
+        """Test the compilation"""
+
+        runtest = self.cfg['runtest']
+        if runtest:
+            if runtest is True:
+                runtest = 'test'
```

Comment:
Are there any options a part from `test`? If that is not the case, I would just hardcode `test` in the `cmd` definition.

Reply:
Yes: You could (ab)use this to pass bazel options: `runtest = '--bazel-opt=foo test'` because bazel differentiates between options before and after the "target" (i.e. "test" here)